src/vos/tests/vts_io.c

```diff
@@ -86,6 +85,41 @@ static struct io_test_flag io_test_flags[] = {
 	},
 };
 
+#define vts_key_gen_helper(dest, len, ukey, lkey, arg)		\
+	do {							\
+		if ((arg)->ofeat & DAOS_OF_##ukey##_UINT64)	\
+			dts_key_gen(dest, len, NULL);		\
+		else						\
+			dts_key_gen(dest, len, (arg)->lkey);	\
+	} while (0)
+
+static void
+vts_key_gen(char *dest, size_t len, bool is_dkey, struct io_test_args *arg)
+{
+	memset(dest, 0, len);
+	if (is_dkey) {
+		vts_key_gen_helper(dest, len, DKEY, dkey, arg);
+	} else if (arg->ofeat & TF_FIXED_AKEY) {
+		if (arg->ofeat & DAOS_OF_AKEY_UINT64) {
```

Comment:
"arg->ofeat & TF_FIXED_AKEY" and "arg->ofeat & DAOS_OF_AKEY_UINT64" cannot be true at the same time, right? You may want to express "else if (arg->ta_flags & TF_FIXED_AKEY)" ?

Reply:
should have been arg->ta_feat & TF_FIXED_AKEY